ksqldb-engine/src/main/java/io/confluent/ksql/execution/json/PlanJsonMapper.java

```diff
@@ -41,7 +41,7 @@
           new KsqlTypesDeserializationModule()
       )
       .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-      .enable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
+      .disable(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES)
```

Comment:
Anyone who has input on if this is a reasonable thing to do, please chime in. It's necessary to make this work with an int as versionNumbers, but if there is a good reason to, we can use Optionals instead.

Reply:
If we go the route of disabling this mapper property, let's just remove the line (in order to have the property disabled by default) rather than explicitly disabling, in order to maintain consistency with the other mappers that also disable the property.